scripts/run_on_popular_crates/src/main.rs

```diff
@@ -0,0 +1,100 @@
+use std::time::Duration;
+use std::collections::HashMap;
+use std::process::Command;
+use std::io::Write;
+use crates_io_api::{CratesQuery, Sort};
+
+/// To get the rustdoc of the baseline, we first create a placeholder project somewhere
+/// with the baseline as a dependency, and run `cargo rustdoc` on it.
+fn create_rustdoc_manifest_for_crate_version(
+    name: &str,
+    version: &str,
+    features: &HashMap<String, Vec<String>>
+) -> cargo_toml::Manifest<()> {
+    use cargo_toml::*;
+
+    Manifest::<()> {
+        package: {
+            let mut package = Package::new(name, version);
+            package.publish = Inheritable::Set(Publish::Flag(false));
+            Some(package)
+        },
+        workspace: Some(Workspace::<()>::default()),
+        lib: {
+            let product = Product {
+                path: Some("lib.rs".to_string()),
+                ..Product::default()
+            };
+            Some(product)
+        },
+        dependencies: {
+            let project_with_features = DependencyDetail {
+                version: Some(version.to_string()),
+                features: features
+                    .iter()
+                    .map(|(key, _values)| key.clone())
+                    .collect(),
+                ..DependencyDetail::default()
+            };
+            let mut deps = DepsSet::new();
+            deps.insert(
+                name.to_string(),
+                Dependency::Detailed(project_with_features),
+            );
+            deps
+        },
+        ..Default::default()
+    }
+}
+
+fn save_manifest(manifest: cargo_toml::Manifest<()>, path: &str) -> anyhow::Result<()> {
+    std::fs::write(path, toml::to_string(&manifest)?)?;
+    Ok(())
+}
+
+fn run_checks_on_manifest() -> anyhow::Result<()> {
+    let output = Command::new("../../../target/debug/cargo-semver-checks")
+        .current_dir("crate_current")
+        .args([
+            "semver-checks",
+            "check-release",
+            "--baseline-root=../crate_baseline/Cargo.toml",
+        ])
+        .output()?;
+    std::io::stdout().write_all(&output.stdout)?;
+    std::io::stdout().write_all(&output.stderr)?;
+    println!();
+    Ok(())
+}
+
+fn main() -> anyhow::Result<()> {
+    let client = crates_io_api::SyncClient::new(
+        "crates_io_api: cargo-semver-checks (obi1kenobi82@gmail.com)",
+        Duration::from_millis(10),
```

Comment:
This is in violation of crates.io's API policy, which states clients should not make requests more than once per second.
```suggestion
        Duration::from_millis(1000),
```

Reply:
Done in 44511f74a70ce7e3527727f08fa7610de5d9fce0.